<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>modal_window</class>
 <widget class="QDialog" name="modal_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>270</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>270</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>270</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>About Converter 2014</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>480</width>
     <height>211</height>
    </rect>
   </property>
   <widget class="QLabel" name="pmTitle">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>60</y>
      <width>150</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Project manager:</string>
    </property>
   </widget>
   <widget class="QLabel" name="pmName">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>60</y>
      <width>150</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Alexey Tyurenkov</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="sgTitle">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>90</y>
      <width>290</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Created by the students of:</string>
    </property>
   </widget>
   <widget class="QLabel" name="sgName_DV">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>130</y>
      <width>150</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Dmitro Vyakhk</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="sgName_IP">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>115</y>
      <width>150</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Ivan Pirus</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="sgName_AS">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>145</y>
      <width>150</width>
      <height>20</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>Andrey Shtadler</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="sgName_AN">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>175</y>
      <width>150</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Andrey Nizinkovskiy</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="sgName_HD">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>160</y>
      <width>150</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Helen Dyogtyeva</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="logo">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>127</width>
      <height>140</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../Img.qrc">:/image/logo.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="progName">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>10</y>
      <width>290</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>ITAInformator</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>40</y>
      <width>290</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="sgTitle_2">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>105</y>
      <width>290</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>IT-Academy</string>
    </property>
   </widget>
   <widget class="QLabel" name="sgTitle_3">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>120</y>
      <width>290</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>group â„–A14</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>230</y>
     <width>70</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../Img.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>modal_window</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>257</x>
     <y>321</y>
    </hint>
    <hint type="destinationlabel">
     <x>258</x>
     <y>169</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
